hunting-buddy.lic

```diff
@@ -52,11 +52,6 @@ class HuntingBuddy
     @hunting_info = format_hunting_info(hunting_info)
   end
 
-  def need_boxes?
-    return false unless @settings.box_hunt_minimum
-    DRCI.count_all_boxes(@settings) <= @settings.box_hunt_minimum
-  end
```

Comment:
Not deleted, just moved down to `over_box_limit?` so the box methods are co-located